tests/trace/langchain/test_tracer.py

```diff
@@ -334,3 +335,62 @@ def test_tracer_retriever_with_exception() -> None:
 
     for span in spans.values():
         assert json_string_to_span(span_to_json(span)) == span
+
+
+@responses.activate
+@pytest.mark.parametrize(
+    "messages",
+    [
+        pytest.param(
+            [
+                ChatMessage(role="system", content="system-message-content"),
+                ChatMessage(role="user", content="user-message-content"),
+                ChatMessage(role="assistant", content="assistant-message-content"),
+                ChatMessage(role="function", content="function-message-content"),
+            ],
+            id="chat-messages",
+        ),
+        pytest.param(
+            [
+                SystemMessage(content="system-message-content"),
+                HumanMessage(content="user-message-content"),
+                AIMessage(content="assistant-message-content"),
+                FunctionMessage(name="function-name", content="function-message-content"),
+            ],
+            id="non-chat-messages",
+        ),
+    ],
+)
+def test_tracing_llm_chat_completions_fails_gracefully(
```

Comment:
I don't think this test is puling its own weight: on one hand it's adding more code for us to understand and maintain, but on the other hand, there's very little risk that someone is going to refactor the code (if ever) in a way that makes it likely to inadvertently forget about the decorator. @mikeldking do you have an opinion?

Reply:
Yeah, honestly I'm not sure we need this PR at all, it's really just for consistency I think.

Reply:
Yeah let's only keep tests that are assets - I think this test only proves that the code works as designed and can be omitted. In general I think tests should strive to mimic real life use-cases so white box testing such as these help individual developer confidence but don't need to live on as part of CI.